etc/systemd/system/zpool@.service.in

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description="Import pool"
+DefaultDependencies=no
+Requires=zpool@%i.path
+After=zpool@%i.path
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStartPre=/sbin/modprobe zfs
+ExecStart=@sbindir@/zpool import -EN %i
+ExecStop=@sbindir@/zpool export %i
+
+[Install]
+WantedBy=local-fs.target
```

Comment:
If you added `WantedBy=zfs-mount.service` and renamed this to `zfs-import-udev@.service.in` then this could be a third pool import option.  Users would then just need to disable the other `zfs-*` services if they didn't want them to use them.  This would allow this same import service to be used for both use cases which would be nice.